example/operator/20-garden.yaml

```diff
@@ -3,4 +3,11 @@ apiVersion: operator.gardener.cloud/v1alpha1
 kind: Garden
 metadata:
   name: garden
-spec: {}
+spec:
+  runtimeCluster:
+    provider:
+      zones:
+      - "0"
+    settings:
+      verticalPodAutoscaler:
+        enabled: true
```

Comment:
We typically put configuration of the deployed system components to `spec.settings` in the `Seed` while fields directly in `.spec` typically contain general specification/configuration for the cluster itself (e.g., provider information like region or zones).
While this might be questionable in general, I think it's better to keep this "in sync". If we think this should be improved we should probably do it for both `Garden` and `Seed` to keep the APIs as much consistent as possible.

Reply:
Ack, let's go with consistency for now